README.md

```diff
@@ -4,23 +4,21 @@ A really great Dota 2 game mode built by [really great people](/contributors.md)
 # Installation
 This addon uses the [Dota 2 Addon Manager](https://github.com/chrisinajar/dota2-addon-manager).
 
-For detailed instructions, check [here](docs/install.md)
+For installation instructions click [here](docs/install.md).
+
+For testing instructions click [here](docs/testing.md).
 
 # Contributing
-Check out the Discord!
+Check out our [Discord](https://discord.gg/WNFBB4d)!
 
 There are also issues marked with [`Great for new contributors`](https://github.com/OpenAngelArena/oaa/issues?q=is%3Aissue+is%3Aopen+label%3A%22great+for+new+contributor%22).
 
-Music can be found at [Soundcloud][soundcloud-link]
+Some useful links with info and documentation can be found [here](docs/useful_links.md).
 
 ### Suggestions?
 Do not create or comment on tickets in this repository for game suggestions. Use the Discord.
 
-### Building npc custom files
-From within the addon directory, run `gulp`.
-
 # License
 MIT
 
-[soundcloud-link]: https://soundcloud.com/OpenAngelArena "Music for Open Angel Arena"
-[discord-link]: https://discord.gg/WNFBB4d "Open Angel Arena Discord Instant Invite"
+[Soundcloud](https://soundcloud.com/OpenAngelArena)
```

Comment:
why rename the variable? this seems like a pointless change that makes this document less standard with the other documents. Also this has nothing to do with gulp.